src/js/lib/agents/encryption.js

```diff
@@ -0,0 +1,75 @@
+const crypto = require('crypto')
+const scryptsy = require('scryptsy')
+
+export default class EncryptionAgent {
```

Comment:
I feel like this class could have a bit of general docs, including maybe a description of `kdfParams`